bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/styles.scss

```diff
@@ -122,4 +122,14 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  border-top: 0 !important;
```

Comment:
Why are you using it !Important over all of these attributes?

Reply:
This was necessary to override the default button style